README.md

```diff
@@ -98,12 +94,10 @@ const validate = validator(schema, {
 
 validate({ hello: 100 });
 console.log(validate.errors)
-// [ { field: 'data["hello"]',
-//     message: 'is the wrong type',
-//     type: 'string',
+// [ { message: 'is the wrong type',
 //     schemaPath: '#/properties/hello',
+//     dataPath: '#/hello',
```

Comment:
Should also update lines 79 - 82 above to reflect changes. 

- `verbose` -> `verboseErrors`
- Mention what `includeErrors` does by itself
- Do you need to have both `verboseErrors` _and_ `includeErrors`?
- Describe `dataPath`

Reply:
Readme needs to be rewritten in general after the doc updates. Let's do that in a separate PR after this and #73, perhaps?

Reply:
I pushed an update for `includeErrors` + `verboseErrors` doc in Readme.

The interaction between those is not final yet.

E.g. we might end up enabling `dataPath` in non-verbose errors if/when that would be made not hurt performance 2x.
I see ways for optimizing that after #73.

Reply:
Upd: partial optimization included here